swc/private/swc.bzl

```diff
@@ -248,6 +278,15 @@ def _swc_impl(ctx):
         inputs.extend(ctx.files.plugins)
         args.add_all(plugin_args)
 
+    if ctx.attr.experimental_emit_isolated_dts:
+        args.add_all(["--config-json", json.encode({
```

Comment:
I assume you tested that `--config-file foo --config-json '{"some": "object"}'` has the desired behavior of overriding the file? 

What happens if the user also gives a `--config-json` in the args they provide?

Reply:
Also would be good to comment if there's an ordering constraint between these two flags

Reply:
These are a few things that I think need more testing, both here and our other `--config-json` cases (see plugins).

Note that we could also go the route where if `experimental_emit_isolated_dts = True` then we assume you set this `emitIsolatedDts` flag yourself, then maybe add some "validation" in the future like tsc does.

IMO if `--config-json` merges+overwrites the existing config then I prefer this instead of requiring validation, but both have trade-offs.